### PR TITLE
Remove duplicated package "gnutls".

### PR DESCRIPTION
**Pull request by iwanami-lin**
# Purpose of pull request

There is no big difference betweein meta-debian and mete-debian-extended. 
So remove bb file and add bbappend to customize its recipe.

# Test
## How to test
Build test
```
$ echo "IMAGE_INSTALL_append += \" gnutls-bin \"" >> conf/local.conf
$ bitbake core-image-minimal
$ bitbake core-image-minimal-sdk -c populate_sdk
```
```
$ bitbake -e gnutls | grep SHRT_VER
# $SHRT_VER
SHRT_VER="3.6"
```
SHRT_VER is an added variable.

## Test result
```
$ MACHINE=qemuarm64 runqemu nographic
```
Referred to http://manpages.ubuntu.com/manpages/xenial/man1/gnutls-serv.1.html
```
$ certtool --generate-privkey > x509-ca-key.pem
$ echo 'cn = GnuTLS test CA' > ca.tmpl
$ echo 'ca' >> ca.tmpl
$ echo 'cert_signing_key' >> ca.tmpl
$ certtool --generate-self-signed --load-privkey x509-ca-key.pem   --template ca.tmpl --outfile x509-ca.pem

$ certtool --generate-privkey > x509-server-key.pem
$ echo 'organization = GnuTLS test server' > server.tmpl
$ echo 'cn = test.gnutls.org' >> server.tmpl
$ echo 'tls_www_server' >> server.tmpl
$ echo 'encryption_key' >> server.tmpl
$ echo 'signing_key' >> server.tmpl
$ echo 'dns_name = test.gnutls.org' >> server.tmpl
$ certtool --generate-certificate --load-privkey x509-server-key.pem   --load-ca-certificate x509-ca.pem --load-ca-privkey x509-ca-key.pem   --template server.tmpl --outfile x509-server.pem

~# gnutls-serv --http --x509cafile x509-ca.pem --x509keyfile x509
-server-key.pem --x509certfile x509-server.pem
Processed 1 CA certificate(s).
HTTP Server listening on IPv4 0.0.0.0 port 5556...done

gnuTLS server started. 
```

### CODE REVIEW

File: recipes-debian/gnutls/gnutls_debian.bbappend [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SHRT_VER = "${@d.getVar('PV').split('.')[0]}.${@d.getVar('PV').split('.')[1]}"
```
Comment (masami256) [inline recipes-debian/gnutls/gnutls_debian.bbappend]: In [poky](https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-support/gnutls/gnutls_3.6.7.bb?h=warrior&id=09dea32f477a3e8855bd35af1c042004a823de8f), SHRT_VER is used for url to source tar file.

```
SHRT_VER = "${@d.getVar('PV').split('.')[0]}.${@d.getVar('PV').split('.')[1]}"

SRC_URI = "https://www.gnupg.org/ftp/gcrypt/gnutls/v${SHRT_VER}/gnutls-${PV}.tar.xz \
           file://arm_eabi.patch \
"
```

However, meta-debian's recipe doesn't use it. so I think remove .bb file is enough.

**Comment by hiraku-wfs** [comment]
@iwanami-lin Why did you close the PR?

**Comment by iwanami-lin** [comment]
I have mistook the operation.

**Comment by hiraku-wfs** [comment]
Okay, I reopen this PR.

**Comment by iwanami-lin** [comment]
thanks.
I fixed a commit.

**Comment by hiraku-wfs** [comment]
Thanks!

**Pull request merged**